tests/bin/echo.test.ts

```diff
@@ -0,0 +1,80 @@
+import * as testUtils from './utils';
```

Comment:
This entire file can likely be removed now, as I believe `echo` has been removed.